### PR TITLE
correct js file path in plugin.xml

### PR DESCRIPTION
**Pull request by Kirkkt**
per https://github.com/BranchMetrics/Smart-App-Banner-Deep-Linking-Web-SDK/issues/196

@dmitrig01

### CODE REVIEW

**Comment by Kirkkt** [comment]
Well, let's just nuke it from web SDK!

@dmitrig01

**Comment by Kirkkt** [comment]
also remove src/cordova

File: release.sh [hunk 1]
```diff
@@ -53,21 +54,7 @@ echo
...
 then
 	sed -i -e "s/\"version\":.*$/\"version\": \"$VERSION_NO_V\",/" bower.json
-fi
-
-read -p "Bump build number? " -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
 	sed -i -e "s/\"build\":.*$/\"build\": \"$VERSION_NO_V\"/" bower.json
-	sed -i -e "s/\"build\":.*$/\"build\": \"$VERSION_NO_V\"/" package.json
```
Comment (dmitrig01) [inline release.sh]: i think we want to keep this line right?

Comment (Kirkkt) [inline release.sh]: I merged it into line 42 so that all package.json changes are together, same for bower.json

Comment (dmitrig01) [inline release.sh]: oops yes

**Comment by dmitrig01** [comment]
:+1:

**Pull request merged**